src/lib/components/Sidebar.svelte

```diff
@@ -30,6 +32,8 @@
         return QRCodeIcon
       case "schemas-icon":
         return SchemasIcon
+      case "dataspace-icon":
```

Comment:
I think this should be refactored to `guides-icon`.

I guess changing this will require a corresponding change in `src/lib/components/navigation.ts` to use the `huides-icon` instead of `dataspace-icon`.

Reply:
... or well, looks like we're adding both icons, so likely we should have a case both for `dataspace-icon` and for `guides-icon`. We'll likely at some point add more dataspace specific things, so I don't see harm in adding support for the icon already, especially since we're adding the SVG in any case.